include/oneapi/mkl/rng/detail/curand/onemkl_rng_curand.hpp

```diff
@@ -82,6 +82,12 @@ ONEMKL_EXPORT oneapi::mkl::rng::detail::engine_impl* create_mrg32k3a(cl::sycl::q
 ONEMKL_EXPORT oneapi::mkl::rng::detail::engine_impl* create_mrg32k3a(
     cl::sycl::queue queue, std::initializer_list<std::uint32_t> seed);
 
+ONEMKL_EXPORT oneapi::mkl::rng::detail::engine_impl* create_mcg59(cl::sycl::queue queue,
```

Comment:
`cl::sycl::` is for SYCL 1.2.1. For SYCL 2020 we can use just `sycl::`

Reply:
I think, as described in issue #149, using the sycl:: namespace is only SYCL conformant when the `sycl/sycl.hpp` header is included.

Reply:
There is a PR from ROCK RAND. Will change after them merge